lib/searchable_select.html.heex

```diff
@@ -95,6 +83,31 @@
               </div>
             </div>
           <% end %>
+          <%= if @limit_hit? && @limit_hit_text do %>
+            <div class="overflow-none shrink-0">
+              <div class="overflow-none shrink-0" id={"#{@id}-remove-limit-option"} phx-click="remove_limit" phx-target={@myself}>
+                <div class="cursor-pointer w-full border-gray-200 border-b hover:bg-primary-400 hover:text-white">
+                  <div class="flex w-full items-center p-2 pl-2 border-transparent border-l-2 relative">
+                    <div class="w-full items-center flex justify-between">
+                      <%= @limit_hit_text %>
```

Comment:
```suggestion
                   <strong><%= @limit_hit_text %></strong>
```
I guess this is personal preference but I'd like it to stand out more as something not quite like the other options:
<img width="471" alt="image" src="https://github.com/Flickswitch/searchable-select/assets/122778195/15b6f5f7-f917-4d9b-aa73-6b964e989015">

vs:
<img width="471" alt="image" src="https://github.com/Flickswitch/searchable-select/assets/122778195/6bb8aa6c-3d97-4444-87d5-ff8bfda31b54">